### PR TITLE
[Opt][VecLoad] Opt the vec stream load performance

### PR DESCRIPTION
**Pull request by HappenLee**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Reduce unless virtual function call and switch case to improve load performance of stream load.

#### 1. Test table:
```
CREATE TABLE `test1` (
  `id` int(11) NOT NULL COMMENT "",
  `k1` varchar(32) NOT NULL COMMENT "",
  `k2` varchar(128) NOT NULL COMMENT "",
  `m` decimal(10, 3) NULL DEFAULT "10.5" COMMENT "",
  `d` date NULL DEFAULT "2014-02-04" COMMENT "",
  `e` datetime NULL DEFAULT "2014-02-04 15:36:00" COMMENT "",
  `f` boolean NOT NULL COMMENT "",
  INDEX index_id (`id`) USING BITMAP COMMENT ''
) ENGINE=OLAP
DUPLICATE KEY(`id`, `k1`)
COMMENT "OLAP"
DISTRIBUTED BY HASH(`id`) BUCKETS 16
PROPERTIES (
"replication_allocation" = "tag.location.default: 1",
"in_memory" = "false",
"storage_format" = "V2"
)
```
#### 2. Test data (1kw)

```
10000,VOtWRdM0vqgMRD4g,QuSHANt7mA0hcsE8O6ggzOMOSLWtmfDs,99.6441922075,1982-11-10,1982-11-10 02:57:53,1
9999,82N10yffLWGu6WM9,zDm0ekBEoYyfFfGGovRgp2hySHw3r31Y,3.9179255308,1992-06-13,1992-06-13 21:54:32,1
9998,To1UUHYi5uy0m0g7,HGihOdi9OyH5LawdgbcgOsGxTaH0du1H,24.4223120552,2007-04-01,2007-04-01 05:15:26,0
9997,zGaXQadwfnchbvhc,XtkA3KuKgK0BuGQONGx4mkZnbm54grSP,47.5443607568,2010-04-26,2010-04-26 03:54:28,0
```
##### 3. opt resutl
|  origin   | opt  |
|  ----  | ----  |
| 25s  | 19s |

## Checklist(Required)

1. Does it affect the original behavior: (No/)
2. Has unit tests been added: (No Need)
3. Has document been added or modified: (No Need)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (Yes)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/vec/exec/vbroker_scanner.cpp [hunk 1]
```diff
@@ -20,11 +20,11 @@
 #include <fmt/format.h>
 
 #include <iostream>
-#include <sstream>
 
 #include "exec/exec_node.h"
 #include "exec/plain_text_line_reader.h"
 #include "exec/text_converter.h"
+#include "exec/text_converter.hpp"
```
Comment (xiepengcheng01) [inline be/src/vec/exec/vbroker_scanner.cpp:27]: can remove `exec/text_converter.h`

**Comment by yixiutt** [comment]
LGTM

File: be/src/vec/exec/vbroker_scanner.cpp (hunk 1 above)
Comment (HappenLee) [reply be/src/vec/exec/vbroker_scanner.cpp:27]: can not compile here

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**